sdk/tests/conformance/reading/read-pixels-test.html

```diff
@@ -307,9 +307,36 @@
       shouldBe("actual", "expected");
     }
     wtu.glErrorShouldBe(gl, gl.NO_ERROR, "there should be no GL errors");
+
+    debug("");
+    debug("check readback into Uint8ClampedArray");
+    continueTestFunc = continueTestPart4;
+    const kSize = 32;
+    canvas.width = kSize;
+    canvas.height = kSize;
+    if (gl.getError() != gl.CONTEXT_LOST_WEBGL) {
+      continueTestPart4();
+    }
+  }
+
+  function continueTestPart4() {
+    const kSize = 32;
+    gl.viewport(0, 0, kSize, kSize);
```

Comment:
clear ignores viewport, fyi!

Reply:
Yeah, was just resetting it since the earlier test set it to a large value.